int32 id
semap_msgs/Polygon3DModel model
---

